docs/auth0_terraform_generate.md

```diff
@@ -5,12 +5,16 @@ has_toc: false
 ---
 # auth0 terraform generate
 
-This command is designed to streamline the process of generating Terraform configuration files for your Auth0 resources, serving as a bridge between the two.
+(Experimental) This command is designed to streamline the process of generating Terraform configuration files for your Auth0 resources, serving as a bridge between the two.
 
 It automatically scans your Auth0 Tenant and compiles a set of Terraform configuration files based on the existing resources and configurations.
 
+Refer to the [instructional guide](https://registry.terraform.io/providers/auth0/auth0/latest/docs/guides/generate_terraform_config) for specific details on how to use this command.
```

Comment:
Link isn't live yet but will be once https://github.com/auth0/terraform-provider-auth0/pull/825 is merged and published.